Add glossaries required for the `osu!catch Ranking Criteria` (#5587)
ppy/osu-wiki: wiki/Glossary/Snapping/en.md

# Snapping

In [osu!catch](/wiki/Game_mode/osu!catch) every [dash](/wiki/Glossary/Dash) and [hyperdash](/wiki/Glossary/Hyperdash) can be clasified into two different snappings, basic-snapped or higher-snapped.

## Basic-snapped

> The term *basic-snapped* is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) whose time between [active objects](/wiki/Glossary/Active_object) is at least twice the time required to allow dashes or hyperdashes, respectively. As an example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic-snapped hyperdash.

Reviewer: ```suggestion
The term **basic-snapped** is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) whose time between [active objects](/wiki/Glossary/Active_object) is at least two times longer than the time required to allow dashes or hyperdashes, respectively. For example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic-snapped hyperdash.
```

## Higher-snapped

> The term *higher-snapped* is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) that doesn't meet the requirement to be a basic-snapped one (i.e. the time between the objects is less than the threshold to be classified as basic).

Reviewer: ```suggestion
The term **higher-snapped** is used for any [dash](/wiki/Glossary/Dash) or [hyperdash](/wiki/Glossary/Hyperdash) that doesn't meet the requirement to be a basic-snapped one (i.e. the time between the objects is shorter than the threshold to be classified as basic).
```
